docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -48,8 +48,20 @@ To use Iceberg, you need:
 General configuration
 ---------------------
 
-These configuration properties are independent of which catalog implementation
-is used.
+To configure the Iceberg connector, create a catalog properties file
+``etc/catalog/myexample.properties`` that references the ``iceberg``
+connector and defines a metastore type, for example, the Iceberg REST catalog: 
+
+.. code-block:: properties
+
+    connector.name=iceberg
+    iceberg.catalog.type=rest
+
+Each metastore catalog type has different :doc:`configuration
```

Comment:
Make this a list to the supported on  .. which is all of them but still..

Reply:
There is already a list with links under requirements (lines 42-44).